core/templates/dev/head/galleries/gallery.html

```diff
@@ -84,7 +84,7 @@
       <button dropdown-toggle type="button" class="btn dropdown-toggle oppia-splash-search-input ng-cloak"
               style="border-bottom-right-radius: 4px; border-top-right-radius: 4px; max-width: 130px;"
               title="<[selectionDetails.languageCodes.description]>">
-        <[selectionDetails.languageCodes.summary | truncate:14 ]>
+        <[selectionDetails.languageCodes.summary | translate:translationData.languages | truncate:14 ]>
```

Comment:
Yep, I meant just the list of languages. I understand the rationale, though -- makes sense! Thanks for explaining.